### PR TITLE
010 runt

### PR DESCRIPTION
**Pull request by vEpiphyte**
* Add Cortex hooks for lifting / editing / deleting runt nodes and properties.
* Add demonstrative runt node implementation in testmodule.
* Add model introspection through storm using runt nodes.
* Remove hooks for `req` on properties. It was a 0.0.x holdover.
* Add a `test:edge` type to the test module.

### CODE REVIEW

File: synapse/datamodel.py [hunk 1]
```diff
@@ -219,6 +221,10 @@ def __init__(self, modl, name, info):
         self.info = info
 
         self.isform = True
+        # self.isrunt = bool(info.get('runt', False))
+        self.isrunt = False
+        if info.get('runt'):
```
Comment (invisig0th) [inline synapse/datamodel.py]:
```
self.isrunt = bool(info.get('runt', False))
```

File: synapse/cortex.py [hunk 2]
```diff
@@ -621,6 +625,69 @@ def onTagDel(self, name, func):
         # TODO allow name wild cards
         self.ontagdels[name].append(func)
 
+    def addRuntLiftHelp(self, form, func):
+        '''
+        Register a lift helper for a given form (and his sub props)
+        '''
+        self._runtLiftFuncs[form.full] = func
+        for name, prop in form.props.items():
```
Comment (vEpiphyte) [inline synapse/cortex.py]: only register a fullly qualified property name here. do not do Form introspection.
